p2p/pre2p/module.go

```diff
@@ -77,6 +78,16 @@ func (m *p2pModule) GetBus() modules.Bus {
 func (m *p2pModule) Start() error {
 	log.Println("Starting network module")
 
+	m.GetBus().
+		GetTelemetryModule().
+		GetTimeSeriesAgent().
+		CounterRegister(
+			pre2pTelemetry.P2P_NODE_STARTED_TIMESERIES_METRIC_NAME,
```

Comment:
The title of the PR says "HotPokt", the description says "raintree" and the code includes both. Which is it?